unlock-js/src/__tests__/walletService.test.js

```diff
@@ -119,13 +117,7 @@ describe('WalletService', () => {
           expect.assertions(2)
           const newAccount = '0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1'
 
-          walletService.web3.eth.accounts.create = jest.fn(() => {
-            return Promise.resolve({
-              address: newAccount,
-            })
-          })
-
-          nock.accountsAndYield([])
+          nock.accountsAndYield([newAccount])
```

Comment:
This is not right. The test actually verifies that when no account is available on the node, one is created by web3... So we cannot have a mock actually return an account.

Reply:
ah that makes sense, I'll add comments for the next victim so we don't forget :)